google/cloud/aiplatform/docker_utils/build.py

```diff
@@ -0,0 +1,513 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import logging
+import os
+from pathlib import Path
+import textwrap
+from typing import List, Optional
+
+from six.moves import shlex_quote
```

Comment:
This is a compatibility library between Python 2 and 3, but we don't support python 2. Please remove this unless there's a compelling reason to keep it.